utils/ccdatofhir/Module.md

```diff
@@ -0,0 +1,83 @@
+  <realmCode code="US"/>
+  <typeId root="2.16.840.1.113883.1.3" extension="POCD_HD000040"/>
+  <templateId root="2.16.840.1.113883.10.20.22.1.1"/>
+  <id extension="EHRVersion2.0" root="be84a8e4-a22e-4210-a4a6-b3c48273e84c"/>
+  <code code="34133-9" displayName="Summary of episode note" codeSystem="2.16.840.1.113883.6.1" codeSystemName="LOINC"/>
+  <title>Medication Summary</title>
+  <recordTarget>
+    <patientRole>
+      <id root="2.16.840.1.113883.19.5" extension="999999999"/>
+      <patient>
+        <name>
+          <given>John</given>
+          <family>Doe</family>
+        </name>
+        <administrativeGenderCode code="M" codeSystem="2.16.840.1.113883.5.1" displayName="Male"/>
+        <birthTime value="1978-06-10"/>
+      </patient>
+    </patientRole>
+  </recordTarget>
+  <component>
+    <structuredBody>
+      <component>
+        <section>
+          <code code="10160-0" codeSystem="2.16.840.1.113883.6.1" codeSystemName="LOINC" displayName="History of Medication Use"/>
+          <title>Medication List</title>
+          <text>
+            <paragraph>Below is a summary of the patient's medication history:</paragraph>
+          </text>
+          <entry>
+            <substanceAdministration classCode="SBADM" moodCode="EVN">
+              <effectiveTime>
+                <low value="2023-05-27"/>
+                <high value="2023-05-29"/>
+              </effectiveTime>
+              <consumable>
+                <manufacturedProduct>
+                  <manufacturedMaterial classCode="MMAT">
+                    <code code="51144-9" codeSystem="2.16.840.1.113883.6.1" codeSystemName="LOINC" displayName="Medication"/>
+                    <name>Medication A</name>
+                  </manufacturedMaterial>
+                </manufacturedProduct>
+              </consumable>
+              <doseQuantity value="1" unit="tablet"/>
+              <routeCode code="26643006" codeSystem="2.16.840.1.113883.6.96" codeSystemName="SNOMED CT" displayName="Oral"/>
+              <administrationUnitCode code="258683007" codeSystem="2.16.840.1.113883.6.96" codeSystemName="SNOMED CT" displayName="Dose"/>
+              <statusCode code="active"/>
+            </substanceAdministration>
+          </entry>
+        </section>
+      </component>
+    </structuredBody>
+  </component>
+</ClinicalDocument>`
+
+public function main() returns error? {
+    // Transform C-CDA document to FHIR R4.
+    // You can pass a C-CDA document and get a FHIR R4 Bundle based on
+    // the mappings defined at
+    // http://build.fhir.org/ig/HL7/ccda-on-fhir/CF-index.htmlr/datatype_maps.html.
+
+    r4:Bundle ccdaToFhirResult = check ccdatofhir:ccdaToFhir(msg);
+    io:println("Transformed FHIR message: ", ccdaToFhirResult.toJson());
+    io:println("------------------------------------------------------------------");
```

Comment:
Is this line necessary?

Reply:
Removed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/817dadb5178bde7d809333a26f465c6688d3879a